tox.ini

```
@@ -36,7 +42,8 @@ deps =
     pylint
     pylint-django
     djangorestframework
-    django-dbdiff
+    ; django-dbdiff
+    git+https://github.com/pfouque/django-dbdiff.git@fix312#egg=django-dbdiff
```

To be rollback after merging/releasing https://github.com/yourlabs/django-dbdiff/pull/21
```suggestion
    django-dbdiff
```